src/webui/www/private/scripts/cache.js

```diff
@@ -76,77 +76,52 @@ window.qBittorrent.Cache ??= (() => {
     class PreferencesCache {
         #m_store = {};
 
-        // obj: {
-        //   onFailure: () => {},
-        //   onSuccess: () => {}
-        // }
-        init(obj = {}) {
-            return fetch("api/v2/app/preferences", {
+        async init() {
+            return await fetch("api/v2/app/preferences", {
```

Comment:
Seems you don't really need the `await` here. Otherwise you'll be waiting for the `fetch` to be resolved which is an unnecessary lag.
```suggestion
            return fetch("api/v2/app/preferences", {
```

Reply:
> You described the behavior and although it is correct, I still feel the `await` is doing unnecessary work. It doesn't really need to resolve it and rewrap it into a promise again.

I assure you, this makes absolutely no difference, even on the slowest of hardware. Both of our time is better spent on shipping new features than figuring out if the extra await should be added or not. 

> I would suggest you ignore the eslint rule instead:

Is this rule providing us any value? Should we get rid of it?

Reply:
>https://eslint.org/docs/latest/rules/no-return-await

Seems I was right about concerning the performance drawback. But JS language fixed/remedied it.

>https://stackoverflow.com/a/44806230

Now it seems `return await ...` does have some nice side-effects... you can keep it as is.